sarxarray/stack.py

```diff
@@ -111,8 +111,8 @@ def _amp_disp(self, chunk_azimuth=500, chunk_range=500):
             {"azimuth": chunk_azimuth, "range": chunk_range, "time": -1}
         )
 
-        amplitude_dispersion = amplitude.std(axis=t_order) / (
-            amplitude.mean(axis=t_order) + np.finfo(amplitude.dtype).eps
+        amplitude_dispersion = amplitude.std(axis=t_order, skipna=False) / (
+            amplitude.mean(axis=t_order, skipna=False) + np.finfo(amplitude.dtype).eps
```

Comment:
by setting `skipna=False`, the useful information in the time series will be ignored. I suggest keeping the default behavior i.e. `skipna=True`. See my comment below.

Reply:
As discussed, we decided to ignore a point when there is nan in the time series. So perhaps we can explain this as in-line comment here.

Reply:
Added more description.